src/nodes/variable_declaration.js

```diff
@@ -30,10 +32,28 @@ export function format(node, context, recur) {
     }
 
     indent && context.indentIn();
+
     for (let i = 0; i < node.declarations.length; i++) {
-        recur(node.declarations[i]);
-        if (node.declarations[i + 1]) {
-            context.write(',\n', context.getIndent());
+        let previous = node.declarations[i - 1];
+        let child = node.declarations[i];
+        let next = node.declarations[i + 1];
+
+        context.write(blockComments.printLeading(child, previous, next));
```

Comment:
you was drunk https://github.com/dmitriiabramov/esfmt/blob/master/src/nodes/object_expression.js#L37 but I feel like a shit to think my own head